library/std/src/f128.rs

```diff
@@ -489,6 +491,15 @@ impl f128 {
     /// assert!(abs_difference <= f128::EPSILON);
     /// # }
     /// ```
+    ///
+    /// Non-positive values:
+    /// ```
+    /// #![feature(f128)]
+    /// # #[cfg(reliable_f128_math)] {
+    /// assert_eq!(0_f128.ln(), f128::NEG_INFINITY);
```

Comment:
Optional nit: all the other examples have a blank line between the feature gate and the first line of the example, i.e. above the `assert_eq`

Reply:
I some blank lines